server/controllers/connectedAccounts.js

```diff
@@ -75,7 +75,7 @@ export const createOrUpdate = (req, res, next, accessToken, data, emails) => {
           const token = user.generateConnectedAccountVerifiedToken(caId, data.profile.username);
           const newLocation = redirect
             ? `${redirect}?token=${token}`
-            : `${config.host.website}/github/apply/${token}?utm_source=${utm_source}`;
+            : `${config.host.website}/github/apply/${token}?utm_source=${utm_source}`; // eslint-disable-line camelcase
```

Comment:
If we rename the variable

```suggestion
            : `${config.host.website}/github/apply/${token}?utm_source=${utmSource}`;
```